wca-regulations.md

```diff
@@ -5,6 +5,11 @@
 
 ## Notes
 
+### Translation
+
+Please note that translations are not official versions (even though they are provided on the WCA website): they may be useful, but they do not come with any guarantees.
```

Comment:
Nit: I know I wrote the original version of this, but I think we can avoid parentheses, e.g.:

How's this?

> Please note that translations are not official versions, even though they are provided on the WCA website for convenience. If there is a difference between a translation and the (current official) English version, the English version must be used.

Reply:
Updated :)